api/src/reviews/review.schema.ts

```diff
@@ -0,0 +1,26 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { HydratedDocument } from 'mongoose';
+
+export type ReviewDocument = HydratedDocument<Review>;
+
+@Schema({ timestamps: true })
+export class Review {
+    id: string;
+
+    @Prop({ required: true })
```

Comment:
Can you replace this with `  @Prop({ type: MongooseSchema.Types.ObjectId, ref: 'User', required: true })`? Find an example in the beacon.schema.